test/unit/pr_checker.test.js

```diff
@@ -300,4 +301,61 @@ describe('PRChecker', () => {
       assert.deepStrictEqual(logger.logs, expectedLogs);
     });
   });
+
+  describe('checkCommitAfterReview', () => {
+    it('should warn about commit pushed since the last review', () => {
+      const logger = new TestLogger();
+      const { commit, review } = commitsAfterReview.singleCommit;
+
+      const expectedLogs = {
+        warn: [
+          [ 'Changes were pushed since the last review:' ],
+          [ 'single commit was pushed after review' ]
+        ],
+        info: [],
+        trace: [],
+        error: []
+      };
+
+      const checker = new PRChecker(logger, {
+        pr: firstTimerPR,
+        reviewers: allGreenReviewers,
+        comments: commentsWithLGTM,
+        reviews: approvingReviews,
+        commits: oddCommits,
```

Comment:
Can you pass the fixtures here instead of passing them into `checkCommitsAfterReview`?